src/shared/github.py

```diff
@@ -108,12 +108,13 @@ def affected_nix_packages() -> str:
 
     def additional_comment() -> str:
         if comment:
+            escaped_comment = comment.replace("`", "\\`")
```

Comment:
This will turn ` into
````
```
\`
```
````
Which renders as
```
\`
```

To escape triple backticks, you need more backticks around the block:

`````
````
```
````
`````
renders as
````
```
````
(yes, I added 5 backticks before and after to show this)